core/tests.py

```diff
@@ -1,3 +1,11 @@
+from core.notifications import EmailSender
 from django.test import TestCase
 
 # Create your tests here.
+class CoreTestCase(TestCase):
+
+    def test_send_email(self):
+        subject = "TestSubject"
+        message = "TestMessage"
+        recipients = ["charlie.gillions@gmail.com"]
```

Comment:
Probably worth removing the hardcoding of this so you don't get spammed every time the tests run XD